var i=0;


try{
  
     console.log(err);
    setTimeout(function(){
                 console.log("callback func")}
                , 0)
    {
        i++;
    }
}catch(err){
    console.log(err);
}